packages/manager/modules/sign-up/src/form/details/details.component.js

```diff
@@ -0,0 +1,24 @@
+import controller from './details.controller';
+import template from './details.html';
+
+export default {
+  name: 'signUpDetails',
+  restrict: 'E',
+  controller,
+  controllerAs: '$ctrl',
+  bindToController: true,
+  template,
+  require: {
+    formCtrl: '^form',
+    signUpFormCtrl: '^signUpForm',
+    signUpDetailsCtrl: 'signUpDetails',
+  },
+  link: (scope, element, attributes, ctrls) => {
+    const controllers = ctrls;
+
+    // add a setFocus method accessible through controller
+    controllers.signUpDetailsCtrl.setElementFocus = (elementName) => {
+      document.getElementsByName(elementName)[0].focus();
+    };
+  },
+};
```

Comment:
This is a `directive` not a `component`
I think you can use the `$postLink` hook with to do this with a component